### PR TITLE
Selenium web driver fix

### PR DESCRIPTION
**Pull request by MarkDeMaria**
@jwforres 

Creates two test pipelines: One for Mac and 1 for Linux.

If you are developer on a Mac, run "grunt test-integration --mac", else run "grunt test-integration" as usual.

### CODE REVIEW

File: .bash_profile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+dd `~/bin` to the `$PATH`
```
Comment (jwforres) [inline .bash_profile]: assume you didnt intend to commit your bash_profile ?

Comment (stevekuznetsov) [inline .bash_profile]: Did you mean to check this in? On a scale from 0 to crazy this is at a solid 8 or 9

File: Gruntfile.js [hunk 2]
```diff
@@ -703,7 +712,7 @@ module.exports = function (grunt) {
       'autoprefixer',
       'connect:test',
       'add-redirect-uri',
-      'protractor:default',
+      (grunt.option('mac') ? 'protractor:mac' : 'protractor:default'),
```
Comment (jwforres) [inline Gruntfile.js]: you need to do the same thing on line 708 above where you see protractor:default

File: test/protractor-mac.conf.js [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+  // Patterns to exclude.
+  exclude: ['integration/e2e.js'], // We are temporarily excluding the e2e tests while we transition to the split merge queue
+
+  // Alternatively, suites may be used. When run without a command line
+  // parameter, all suites will run. If run with --suite=smoke or
+  // --suite=smoke,full only the patterns matched by the specified suites will
+  // run.
+  suites: {
+    // smoke: 'spec/smoketests/*.js',
+    rest_api: 'integration/rest_api/*.js' // This suite of tests should only require a running master api, it should not require a node
```
Comment (jwforres) [inline test/protractor-mac.conf.js]: i think the suites are updated in master you should compare this to the main protractor.conf in master and make sure this is the same list

**Comment by MarkDeMaria** [comment]
@jwforres Changes made.

File: test/protractor-mac.conf.js [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+  // Patterns to exclude.
+  exclude: ['integration/e2e.js'], // We are temporarily excluding the e2e tests while we transition to the split merge queue
+
+  // Alternatively, suites may be used. When run without a command line
+  // parameter, all suites will run. If run with --suite=smoke or
+  // --suite=smoke,full only the patterns matched by the specified suites will
+  // run.
+  suites: {
+    // smoke: 'spec/smoketests/*.js',
+    'add-to-project': 'integration/features/user_adds_template_to_project.spec.js',
```
Comment (jwforres) [inline test/protractor-mac.conf.js]: sorry this actually needs to be updated again because of changes that went in for this PR #1195

**Comment by MarkDeMaria** [comment]
@jwforres Changes made!

**Comment by jwforres** [comment]
[merge]

**Pull request merged**